docs/running-in-production-dockerized.md

```diff
@@ -0,0 +1,88 @@
+# Running in Production Dockerized
+
+As the PrairieLearn source code is publicly-available, it's possible to run PrairieLearn on your own infrastructure. Running a single instance of the PrairieLearn server may be appropriate for tens or hundreds of total users, and a number of universities have done this successfully.
+
+## Getting started
+
+Follow the steps to [install PrairieLearn with local source code](./installingLocal.md). Then run this command in the root folder:
+```sh 
+docker-compose -f docker-compose-production.yml up
+```
+Then access PrairieLearn from port `3000`.
+
+## Configuration
+
+PrairieLearn can be configured by a `config.json` in the root of the repository.
+
+- First make the file `config.json` in your root repository.
+- Add the following line to `docker-compose-production.yml` under `volumes`:
+
+```sh
+- ./config.json:/PrairieLearn/config.json
+```
+
+The `config.json` file should contain appropriate overrides for the keys in [`lib/config.js`](`https://github.com/PrairieLearn/PrairieLearn/blob/master/lib/config.js`). At a minimum, you'll probably want to update the various `postgres*` options to point it at your database.
+
+## Reverse Proxy
+
+A reverse proxy can be implemented using something like [Apache](https://httpd.apache.org/docs/2.4/howto/reverse_proxy.html) or [NGINX](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/).
+
+In `config.json` to configure your domain add:
+
+```json
+{
+"serverCanonicalHost": "https://yourdomain"
+}
+```
+
+## Authentication
+
+PrairieLearn currently has 4 ways to do user authentication. Read more at [authentication](./authentication.md).
+
+## Admin User

```

Comment:
This core information here (adding an admin user) is broadly useful to folks whether or not they're running PL inside of Docker. I'd like to see that info moves somewhere else that we can link to from both the Docker and non-Docker version of this documentation.

Reply:
Moved to file `admin-user.md`